### DISCUSSION TITLE
What are the diffrent between these two regex?

### DISCUSSION THREAD

**Discussion by Jesan71** [Difegue/LANraragi#810] [Q&A]
In metadata plugin RegexParse.pm
![图片](https://github.com/Difegue/LANraragi/assets/76234770/100ccc60-3005-4702-bec3-4caa49d4644e)
I do know that `(\(([^([]+)\))?` is used to match content enclosed in ( ) and `(\[([^]]+)\])?` is used to match content enclosed in [ ]. But why they use different excluded characters?

**Comment by Difegue** [comment]
👋 i wrote this regex 8 years ago and have absolutely no idea anymore

**Reply by Jesan71** [reply]
All right. It's not much important lol. Thanks for your work.
